eng/pipelines/runtime.yml

```diff
@@ -495,6 +498,8 @@ extends:
           nameSuffix: _Threading_PerfTracing
           extraBuildArgs: /p:MonoWasmBuildVariant=perftrace
           alwaysRun: ${{ variables.isRollingBuild }}
+          scenarios:
+            - normal
```

Comment:
This is using `wasm-build-only` which won't be running any tests, so `scenarios` won't be used.

Reply:
Oh, oops, I added that `scenarios: normal` block to the wrong leg. I didn't want to touch perftrace in this PR.